### PR TITLE
Add Vale terms based on feedback from KUDO for Kubeflow

### PR DESCRIPTION
**Pull request by ianhellstrom**
Minimize false positives by adding more acceptable words and names.

### CODE REVIEW

File: .github/styles/Vocab/Docs/accept.txt [hunk 1]
```diff
@@ -1,24 +1,64 @@
-D2iQ
-COPS
-Mesos
+addon
+Alertmanager
+Ansible
+autoscal(s|ing)
+Cassandra
+checkpoint(s|ed|ing)
```
Comment (porridge) [inline .github/styles/Vocab/Docs/accept.txt:6]:
```suggestion
checkpoint(|s|ed|ing)
```
We want to allow `checkpoint` as well, right? Or is that accepted already?

**Review by porridge — APPROVED** [review]

File: .github/styles/Vocab/Docs/accept.txt (hunk 1 above)
Comment (ianhellstrom) [reply .github/styles/Vocab/Docs/accept.txt:6]: Good point!

Comment (ianhellstrom) [reply .github/styles/Vocab/Docs/accept.txt]: I think that should already be a word on and of itself but the verb conjugations may not be. Still, I like making it clear that all versions are accepted.

File: .github/styles/Vocab/Docs/accept.txt [hunk 2]
```diff
@@ -1,24 +1,67 @@
-D2iQ
-COPS
-Mesos
+addon
+Airflow
+Alertmanager
+Ansible
+autoscal(s|ed|ing)
```
Comment (cmastr) [inline .github/styles/Vocab/Docs/accept.txt:5]: I don't think autoscal is a word

**Review by cmastr — CHANGES_REQUESTED** [review]

File: .github/styles/Vocab/Docs/accept.txt (hunk 2 above)
Comment (porridge) [reply .github/styles/Vocab/Docs/accept.txt]: Pushed https://github.com/mesosphere/dcos-docs-site/pull/3214/commits/8068e2bfa9ed2e3b6d38baa1597e764942857d45 to fix.

**Review by cmastr — APPROVED** [review]

**Comment by porridge** [comment]
Resolved the conflict and made sure the terms are sorted @cmastr

**Review by cmastr — APPROVED** [review]

**Pull request merged**